tests/pact/index.pact.spec.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const { pactWith } = require('jest-pact')
+const { Hello_Greet } = require('../../ui/client/hello/index')
```

Comment:
Codacy found an issue: [Identifier 'Hello_Greet' is not in camel case.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5869027)